spikeextractors/tests/TestData.clu

```diff
@@ -0,0 +1,72 @@
+8
```

Comment:
can you remove these test files? (See my comments below)

Reply:
Done.